diff --git a/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/utils/ViewTagResolver.kt b/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/utils/ViewTagResolver.kt
index 54101e8..7b165bf 100644
--- a/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/utils/ViewTagResolver.kt
+++ b/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/utils/ViewTagResolver.kt
@@ -32,7 +32,9 @@ open class ViewTagResolver(val context: ReactApplicationContext) {
                 try {
                     val view = manager.resolveView(viewTag)
 
-                    list.forEach { it.fn(view) }
+                    if (view != null) {
+                        list.forEach { it.fn(view) }
+                    }
                 } catch (err: IllegalViewOperationException) {
                     list.forEach { it.reject?.reject(err) }
                 }
